ui/src/components/credentials/attributes/Boolean.tsx

```diff
@@ -15,6 +15,7 @@ export function Boolean({
       label={<Typography.Text>{attribute.schema.title || attribute.name}</Typography.Text>}
       name={["credentialSubject", ...parents.map((parent) => parent.name), attribute.name]}
       required={attribute.required}
```

Comment:
You don't need this flag if you use `rules`. But why do you want to do this, adding generic validation messages?

Reply:
But we are letting the BE and parsers flag errors, not relying on inline form errors. If you define `rules`, it always defaults to a message.

![Screenshot from 2023-04-26 19-17-27](https://user-images.githubusercontent.com/2691584/234653068-e444b0d3-6b52-4e05-ac52-fa714553dea4.png)

Reply:
The problem with the errors from the backend is that are not targeted to the fields, then, do you want to remove all validation and let the user be aware of the errors through toast bar notifications? or are you asking for customized inline errors? I'm a bit lost.

Reply:
> let the user be aware of the required field errors through toast bar notifications

This is how we always had it. If we want to add extra form validation, then it should be everywhere (replacing the `required` flag in `Form.Item` with `rules`). The problem is that it's [buggy](https://github.com/0xPolygonID/sh-id-platform/pull/342#discussion_r1177519945), which is why I would avoid adding it right now also in the interest of time.

I recommend adding an issue to consolidate all `required`, `requiredMark` and `rules` so that form validation can be implemented properly.

Reply:
Done as agreed @cneuro.

Reply:
Relevant issue: https://polygonid.atlassian.net/browse/PID-651